### PR TITLE
Most occurred element of an array #755 Done

### PR DESCRIPTION
**Pull request by krishna-NIT**
@nakul-19 @rudrakshi99 @tarun26091999 please review it
#755  Done in Java
@OjusWiZard

### CODE REVIEW

**Comment by krishna-NIT** [comment]
@yasharth291 Please Review my Code

**Comment by krishna-NIT** [comment]
@yasharth291 Please review it because Daily I get Conflicts in Readme File, I have to Resolve them Daily, Am I making any Mistake in Readme ?, can you please tell me why I am Getting Conflicts in Readme always, coz I am new to GIT and GITHUB

**Comment by krishna-NIT** [comment]
@plazzy99 please Review it and please tell me if any changes are required

**Comment by krishna-NIT** [comment]
@rish-singhal LGTM
Sir Hamare bhi dekh lo

File: Code/Java/mostoccured.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    Test Cases:
+        Input : 5
+                8 9 8 2 4 1
+        Output: 8
+
+        Input : 6
+                2 4 6 5 4 3 4
+        Output: 4
+
+        Time Complexity: O(n)
```
Comment (rish-singhal) [inline Code/Java/mostoccured.java:79]: Describe n here

**Review by rish-singhal — CHANGES_REQUESTED** [review]

File: Code/Java/mostoccured.java (hunk 1 above)
Comment (krishna-NIT) [reply Code/Java/mostoccured.java]: done sir

**Comment by krishna-NIT** [comment]
lol, mera hi bhul gaya

**Pull request merged**